docs/schema/2.0/manifest.schema.json

```diff
@@ -177,12 +164,30 @@
               "type": "string"
             }
           }
-        },
-        "publicKeys": {
-          "type": "array",
-          "items": {
-            "type": "string",
-            "format": "uri"
+        }
+      }
+    },
+    "apiS2S": {
+      "scopes": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "s2sId": {
```

Comment:
Is this also going to be renamed to `guid`?

Reply:
Yes, seems I missed it. Thank you.